<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>员工管理系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/img/Logo.png</normaloff>:/img/Logo.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2" rowstretch="2,12,2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="titleLayout"/>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="centerLayout"/>
     </item>
     <item row="2" column="0">
      <layout class="QVBoxLayout" name="statueLayout">
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
